ocs_ci/utility/ibmcloud.py

```diff
@@ -337,6 +338,52 @@ class IBMCloud(object):
     Wrapper for Ibm Cloud
     """
 
+    def start_nodes(self, nodes, wait=True):
+        """
+        Start nodes on IBM Cloud.
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): True for waiting the instances to start, False otherwise
```

Comment:
Add `Raises` section mention: `ValueError` there as this is one you raises in some cases ;)